katdal/__init__.py

```diff
@@ -14,32 +14,40 @@
 # limitations under the License.
 ################################################################################
 
-"""KAT data access library to interact with HDF5 and MeasurementSet files.
+"""KAT data access library for HDF5 files, MeasurementSets and chunk stores.
 
 Overview
 --------
 
-This module serves as a data access library to the HDF5 files produced by the
-Fringe Finder and KAT-7 data capturing systems. It uses memory carefully,
-allowing files to be inspected and partially loaded into memory. Data sets may
-be concatenated and split via a flexible selection mechanism. In addition, it
-provides a script to convert these HDF5 files to CASA MeasurementSets.
+This module serves as a data access library to interact with the chunk stores
+and HDF5 files produced by the MeerKAT radio telescope and its predecessors
+(KAT-7 and Fringe Finder). It uses memory carefully, allowing data sets to be
+inspected and partially loaded into memory. Data sets may be concatenated and
+split via a flexible selection mechanism. In addition, it provides a script to
+convert these data sets to CASA MeasurementSets.
 
 Quick Tutorial
 --------------
 
-Open any HDF5 file through a single function to obtain a data set object::
+Open any data set through a single function to obtain a data set object::
 
   import katdal
   d = katdal.open('1234567890.h5')
 
-This automatically determines whether it is a version 1 (FF) or version 2
-(KAT-7) file. Multiple files (even of different versions) may also be
-concatenated together (as long as they have the same dump rate)::
+This automatically determines the version and storage location of the data set.
+The versions roughly map to the various instruments::
+
+  - v1 : Fringe Finder / FF (HDF5 file)
+  - v2 : KAT-7 (HDF5 file)
+  - v3 : RTS / AR1 (HDF5 file)
```

Comment:
Same comment as above. Do we really need two copies of it, or can we just direct people to read the README for the overview?

Reply:
I want to be able to do `katdal?` in IPython. The ideal would be to load README into the docstring on demand.